tests/framework/common_profile.go

```diff
@@ -123,6 +125,11 @@ func (sd *DataHandle) Iterate(f func()) {
 		sd.OutputIteration(sd.Iterations, os.Stdout)
 		fmt.Println("--------")
 
+		restarts := Td.VerifyRestarts()
+		if restarts {
+			Fail("Failed at it %d, seen restarts", sd.Iterations)
```

Comment:
I wasn't able to understand this error message. What does "Failed at it" refer to?

Reply:
"at iteration" :) 
want me to clarify the log line in code?